rails-commenting.rb

```diff
@@ -48,15 +48,15 @@ def update
   def destroy
     @post = BlogPost.find(params[:id])
     if @post.destroy
-      # ---8)
+      # ---8) This line will redirect to, or display blog_posts_path
       redirect_to blog_posts_path
     end
   end
 
-  # ---9)
+  # ---9) Anything under this line will not be displayed to users
```

Comment:
The private keyword is used to make anything below it only available in the scope of this controller